openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -602,10 +602,14 @@ Token Responses are made as defined in [@!RFC6749].
 
 In addition to the response parameters defined in [@!RFC6749], the AS MAY return the following parameters:
 
-* `c_nonce`: OPTIONAL. JSON string containing a nonce to be used to create a proof of possession of key material when requesting a Credential (see (#credential_request)). When received, the Wallet MUST use this nonce value for its subsequent Credential Requests until the Credential Issuer provides a fresh nonce.
+* `c_nonce`: OPTIONAL. JSON string containing a nonce to be used when creating a proof of possession of the key proof (see (#credential_request)). When received, the Wallet MUST use this nonce value for its subsequent requests until the Credential Issuer provides a fresh nonce.
 * `c_nonce_expires_in`: OPTIONAL. JSON integer denoting the lifetime in seconds of the `c_nonce`.
+* `authorization_details`: REQUIRED when `authorization_details` parameter is used to request issuance of a certain Credential type as defined in (#authorization-details). A JSON array of objects as defined in Section 7 of [@!RFC9396]. This specification defines the following parameter to be used with authorization details type `openid_credential` in the Token Response:
+  * `c_instance_identifiers`: OPTIONAL. JSON array of JSON strings that each uniquely identify a Credential instance that can be issued using Access Token returned in this response. Each Credential instance can be a unique Credential of the same format and type, but with different claim values or different subset of claims within the claimset identified by the Credential type. This parameter can also be used to simplify the Credential Request, since as defined in (#credential_request) `c_instance_identifier` parameter replaces `format` and any other Credential format specific parameters in the Credential Request. When received, the Wallet MUST use these values together with an Access Token in the subsequent Credential Request(s).
```

Comment:
Detail: I'm not 100% comfortable with the `instance_identifier` naming, namely because:
- The credential was not yet created (that will only happen after a credential request), so there isn't yet an instance, right?
- Also, this identifier may be confused with the `callback_id` from https://github.com/openid/OpenID4VCI/pull/70/files, which is described as `A JSON string identifying an issued Credential`.

Would `credential_request_identifier`(or an abbreviation of it) be a better name , since it provides an identifier to request a specific thing on the credential request.

Reply:
during SIOP call on the week of Oct 16th, we discussed that a term instance does not necessarily mean after issuance and can identify credentials before they are being issued.

Reply:
solved by renaming to credential_identifier for now. cc issue https://github.com/openid/OpenID4VCI/issues/91